sound/soc/sof/core.c

```diff
@@ -394,6 +394,9 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 
 	sof_set_fw_state(sdev, SOF_FW_BOOT_NOT_STARTED);
 
+	/* initialize ipc ops */
+	sdev->ipc_ops = &ipc3_ops;
```

Comment:
Would it make sense to put the ops under sdev->ipc ?

Reply:
done